app/services/notification/AWSMailClient.scala

```diff
@@ -0,0 +1,46 @@
+package services.notification
+
+import javax.inject.Inject
+
+import com.amazonaws.handlers.AsyncHandler
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailServiceAsync
+import com.amazonaws.services.simpleemail.model._
+import models.{Instance, Owner}
+import play.api.libs.concurrent.Execution.Implicits._
+import play.api.{Configuration, Logger}
+
+import scala.concurrent.{Future, Promise}
+
+class AWSMailClient @Inject()(amazonMailClient: AmazonSimpleEmailServiceAsync, configuration: Configuration) {
+
+  def send(owner: Owner, instances: Seq[Instance]): Future[String] = {
+    val request: SendEmailRequest = createEmailRequest(owner, instances)
+
+    val promise = Promise[SendEmailResult]()
+    val responseHandler = new AsyncHandler[SendEmailRequest, SendEmailResult] {
+      override def onError(e: Exception): Unit = {
+        Logger.warn(s"Could not send mail: ${e.getMessage}\n" +
```

Comment:
Oh I misread this the first time and thought it was putting the email in the error! Yeah, what Simon said :-)

Sorry!